test/nntrainer_test_util.cpp

```diff
@@ -218,6 +218,30 @@ makeGraph(const std::vector<LayerRepresentation> &layer_reps) {
   return graph_rep;
 }
 
+nntrainer::GraphRepresentation
+makeGraph_V2(const std::vector<LayerRepresentation> &layer_reps) {
+  static auto &ac = nntrainer::AppContext::Global();
+
+  nntrainer::GraphRepresentation graph_rep;
+  auto model_graph = nntrainer::NetworkGraph();
+  for (auto &layer_representation : layer_reps) {
+    std::shared_ptr<nntrainer::LayerNode> layer = nntrainer::createLayerNode(
+      ac.createObject<nntrainer::Layer>(layer_representation.first),
+      layer_representation.second);
+    model_graph.addLayer(layer);
+  }
+  // compile with loss
+  model_graph.compile("mse");
```

Comment:
What if user already added loss layer?

Reply:
no. It is only for the test case and It is the only way to define the loss layer is by the defining model property in normal case.

Reply:
If it will be only used in unittest then I think reside in util.cpp is not a good way I guess?
I agreed that most of user will defining the loss by passing the loss type to model property but we allowed it to define by layer aren't we?

Reply:
Why it is not good? This is the code for the test only. I don't agree with you saying good way. Any case it needs compiled connection whose inputs do not follow the loss layer.. then it might be help. Besides, if the user define the loss with model property and set the loss layer.. it causes the error and the same is here.

Reply:
Oh sorry. I was confused.
I think the file is util_func.cpp but it was in test_unit.cpp actually.
Most of my comments coming from this confusion. My bad.